THEME.md

```diff
@@ -205,25 +205,46 @@ It's a good idea to add this to the top of the template to improve the styling o
 
 ## Action Injection
 
-The following will inject the action link (or button) into the e-mail:
+The following will inject the action link (or button) into the e-mail.
 
 ```html
 <!-- Action -->
+<% if(locals.action.constructor !== Array){ %>
+<%    action = [action]; %>
+<% } %>
+
 <% if (locals.action) { %>
-    <p><%- action.instructions %></p>
-    <a href="<%- action.button.link %>" style="color:<%- action.button.color %>" target="_blank">
-        <%- action.button.text %>
-    </a>
+    <% action.forEach(function (actionItem) { -%>
+        <p><%- actionItem.instructions %></p>
+        <a href="<%- actionItem.button.link %>" style="color:<%- actionItem.button.color %>" target="_blank">
+            <%- actionItem.button.text %>
+        </a>
+    <% }) -%>
 <% } %>
 ```
 
 It's a good idea to add this to the top of the template to specify a fallback color for the action button in case it wasn't provided by the user:
 
 ```html
 <%
+if (locals.action){
+
+    // If there is only one action, make into array of one actions
+    if(locals.action.constructor !== Array){
+        action = [action];
+    }
+
+    locals.action.forEach(function(actionItem){
+        fallbackColor(actionItem.button);
```

Comment:
I think that we should omit the function `fallbackColor` and move its logic to within the `forEach` to simplify the code. What do you think?